app/src/main/java/com/example/bigowlapp/activity/SendSmsInvitationActivity.java

```diff
@@ -28,7 +29,7 @@ protected void onCreate(Bundle savedInstanceState) {
         String contactDetails = getIntent().getStringExtra("contactDetails");
         String contactNumber = getIntent().getStringExtra("contactNumber");
 
-        noteText = "Contact: " + contactDetails + " is not yet registered to the application. Send her/him this invitation text sms";
+        noteText = "Contact: " + contactDetails + " is not yet registered to the application. Send her/him the bellow invitation by text sms";
```

Comment:
typo, also better wording would be "Send them the invitation below by text SMS."

Reply:
I don't think we should say "them" as it is one contact
I will fix the typo

Reply:
her/him is fine too. Them and her/him are both appropriate English. main thing would be just fixing the typo really.